python/doc/examples/probabilistic_modeling/quick_start_guide_distributions.ipynb

```diff
@@ -0,0 +1,824 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create univariate and multivariate distributions: a quick start guide to distributions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Abstract \n",
+    "\n",
+    "In this example, we present the classes which create univariate and multivariate distributions and present how the library is a probabilistic programming library. For the univariate distributions, we show how to compute the probability density, the cumulated probability distribution and the quantiles. We also show how to create graphics. The `ComposedDistribution` class, which creates a distribution based on its marginals and its copula, is presented. We show how to truncate any distribution with the `TruncatedDistribution` class."
```

Comment:
Here are a few suggestions, but I think further changes should be made. The main issue is that it is not clear what the phrase "probabilistic programming library" means until the next paragraph. Not sure how to do it better though.
```suggestion
    "In this example, we present classes for univariate and multivariate distributions. We demonstrate the probabilistic programming capabilities of the library. For univariate distributions, we show how to compute the probability density, the cumulated probability density and the quantiles. We also show how to create graphics. The `ComposedDistribution` class, which creates a distribution based on its marginals and its copula, is presented. We show how to truncate any distribution with the `TruncatedDistribution` class."
```

Reply:
What I mean by "probabilistic programming" is: 
https://en.wikipedia.org/wiki/Probabilistic_programming 
Fixed.